recipe/emailpassword/epmodels/models.go

```diff
@@ -26,6 +27,7 @@ type TypeNormalisedInput struct {
 	ResetPasswordUsingTokenFeature TypeNormalisedInputResetPasswordUsingTokenFeature
 	EmailVerificationFeature       evmodels.TypeInput
 	Override                       OverrideStruct
+	GetEmailDeliveryConfig         func() emaildelivery.TypeInputWithService
```

Comment:
This function should take Recipe interface and everything else that it needs to make the backwards compatible service